Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-jdk-alpine
+VOLUME /tmp

```

Comment:
There is a specific need to specify this volume?

Reply:
Maybe the application could use it to store temporary files. I don't know, just a guess

Reply:
Ok, let's leave that VOLUME in place for now